fswatch/src/fswatch.cpp

```diff
@@ -27,14 +27,14 @@
 #include <vector>
 #include <array>
 #include <map>
-#include "libfswatch/c++/path_utils.hpp"
-#include "libfswatch/c++/event.hpp"
-#include "libfswatch/c++/monitor.hpp"
-#include "libfswatch/c++/monitor_factory.hpp"
-#include "libfswatch/c/error.h"
-#include "libfswatch/c/libfswatch.h"
-#include "libfswatch/c/libfswatch_log.h"
-#include "libfswatch/c++/libfswatch_exception.hpp"
+#include "path_utils.hpp"
```

Comment:
Using the library prefix was a conscious choice. Why would this refactor be required?